monai/transforms/utils.py

```diff
@@ -368,6 +369,69 @@ def check_non_lazy_pending_ops(
         warnings.warn(msg)
 
 
+def map_and_generate_sampling_centers(
+    label: NdarrayOrTensor,
+    spatial_size: Sequence[int] | int,
+    num_samples: int,
+    label_spatial_shape: Sequence[int],
+    indices: Sequence[NdarrayOrTensor],
```

Comment:
Consider removing the input `indices: Sequence[NdarrayOrTensor],` or modifying it to `indices: Sequence[NdarrayOrTensor] | None = None,` because the function` map_classes_to_indices `used in the program takes the input` label` and returns` indices`.